prism/src/main/java/org/apache/falcon/plugin/GraphiteNotificationPlugin.java

```diff
@@ -50,10 +50,9 @@ public void monitor(ResourceMessage message) {
             String entityName = StringUtils.isNotBlank(message.getDimensions().get("entityName"))
                     ? message.getDimensions().get("entityName") :message.getDimensions().get("entity-name");
             String prefix = StartupProperties.get().getProperty("falcon.graphite.prefix");
-            String separator = ".";
             LOG.debug("message:" + message.getAction());
             if (entityType.equalsIgnoreCase(EntityType.PROCESS.name())) {
-                Entity entity = ConfigurationStore.get().get(EntityType.PROCESS, entityName);
```

Comment:
This change to fix FALCON-719 is not consistent with the change made in trunk. Please match the change with that of trunk.